docs/node-cleanup-controller.md

```diff
@@ -0,0 +1,44 @@
+# Local Volume Node Cleanup Controller
```

Comment:
I would also add a short description and link to here from the top level README in the overview section.